libs/blocks/merch-card/merch-card.js

```diff
@@ -109,10 +109,10 @@ const MINI_COMPARE_CHART = 'mini-compare-chart';
 
 const MULTI_OFFER_CARDS = ['plans', 'product', MINI_COMPARE_CHART];
 // Force cards to refresh once they become visible so that the footer rows are properly aligned.
-const sectionObserver = new MutationObserver((mutations) => {
-  mutations.forEach((mutation) => {
-    if (mutation.type === 'attributes' && mutation.attributeName === 'data-status') {
-      const container = mutation.target.closest('main > div');
+const sectionObserver = new IntersectionObserver((entries) => {
```

Comment:
can you rename to `intersectionObserver` and call `unobserve` (https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/unobserve) after adjusting the height.

Reply:
fixed

Reply:
I also removed the object params which are not needed when calling observe in L399